arch/lib/Makefile

```diff
@@ -155,7 +157,7 @@ define cmd_lib_bounds
 	 echo "#define GENERATED_BOUNDS_H"; \
 	 echo ""; \
 	 echo "#define NR_PAGEFLAGS (__NR_PAGEFLAGS)"; \
-	 echo "#define MAX_NR_ZONES (__MAX_NR_ZONES)"; \
+	 echo "#define MAX_NR_ZONES 3"; \
```

Comment:
I see. Thanks. I made a new pull request. I have solved the coding style
errors reported
by checkpatch.pl script. This time the code should be cleaner.

I think the next step is to use x86 headers instead of arm header. I will
start to work on this.

I also have a long-term puzzle in my mind. Is it possible to make kswapd
work in Lib OS.
If kswapd can be ported to user land, all mm code can be ported without any
problem.

Have a nice day!

Yizheng







On Sun, Sep 6, 2015 at 10:54 PM, Hajime Tazaki <notifications@github.com>
wrote:

> In arch/lib/Makefile
> <https://github.com/libos-nuse/net-next-nuse/pull/47#discussion_r38831044>
> :
>
> > @@ -155,7 +157,7 @@ define cmd_lib_bounds
> >  	 echo "#define GENERATED_BOUNDS_H"; \
> >  	 echo ""; \
> >  	 echo "#define NR_PAGEFLAGS (__NR_PAGEFLAGS)"; \
> > -	 echo "#define MAX_NR_ZONES (__MAX_NR_ZONES)"; \
> > +	 echo "#define MAX_NR_ZONES 3"; \
>
> For this problem, I don't understand how to put the hard-coded value in
> .config file.
>
> for instance, you added PAGE_OFFSET value in arch/lib/Kconfig. this value
> would be written in .config after make defconfig.
>
> config PAGE_OFFSET
>     hex
>     default 0xC0000000
>
> you can refer this value by using CONFIG_PAGE_OFFSET by #include .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/libos-nuse/net-next-nuse/pull/47/files#r38831044>.
>